### ISSUE TITLE
Insomnia show error when ResponseEval's Javascript code contain ' or "

### ISSUE THREAD

**Issue by TuyenTk** [taylorgoolsby/insomnia-plugin-response-eval#2]
Using Insomnia to get response from following URL: https://jsonplaceholder.typicode.com/todos

![image](https://user-images.githubusercontent.com/4112561/75085379-0c999c80-555b-11ea-9cf3-2b4d3c60a6cb.png)

Then in onther request, I use ResponseEval to get ID of the ToDo item with title **delectus aut autem**:

```
`${JSON.parse(output).filter(m => m.title == 'delectus aut autem')[0].id}`
```
Insomnia show this error:

![image](https://user-images.githubusercontent.com/4112561/75085130-b4fa3180-5558-11ea-8f7b-36dd83fd0c57.png)

If I change `'` to `"`:

```
`${JSON.parse(output).filter(m => m.title == "delectus aut autem")[0].id}`
```

I get this error:

![image](https://user-images.githubusercontent.com/4112561/75085157-e4a93980-5558-11ea-96f6-91b3b7d78f27.png)

When I hover above the tag, I see the value `1`, and I can send the request with that value.

**Comment by taylorgoolsby** [comment]
I'm not able to reproduce either of those errors. I tried both using `"` and `'`, and they both worked.
![image](https://user-images.githubusercontent.com/14286600/75088383-4d64c780-5501-11ea-9525-a7596624411c.png)
![image](https://user-images.githubusercontent.com/14286600/75088384-548bd580-5501-11ea-9d91-c581f276aee9.png)

**Comment by taylorgoolsby** [comment]
BTW, you can use insomnia's built in `response` tag for your purpose, like this:
![image](https://user-images.githubusercontent.com/14286600/75088436-0aefba80-5502-11ea-9fc4-601fa038a1f5.png)

**Comment by TuyenTk** [comment]
Hi @taylorgoolsby 
As I said:

> When I hover above the tag, I see the value 1, and I can send the request with that value.

You tried to reproduce it, only to miss one last step when the error occur. Your plugin done the job of eval Java script. The error is not appear when you input parameters. But when you press Done, Insomnia show error in the editor. Please input a json like I do you' ll see it.

By the way I'm aware of Json Path. But in one of my API I need caculate sum of 3 Json Path's value. Then I found you plugin very handy. ResponseEval work. Only Insomnia show error in tbe editor.

**Comment by taylorgoolsby** [comment]
I think this is an error with Insomnia. 
Even when using the built-in `response` tag, I see an error.
![image](https://user-images.githubusercontent.com/14286600/75120918-527e5f80-5644-11ea-92a2-0ac5d6c20063.png)

It appears `response-eval` is working even though there is an error in Insomnia's text field.
![image](https://user-images.githubusercontent.com/14286600/75120907-2ebb1980-5644-11ea-88a7-c886a286f454.png)

**Comment by taylorgoolsby** [comment]
Fixed in abe98a27c62600618101fff22492d39627a7b600
